### PR TITLE
Docking station - update test matrix and hardware matrix

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki [kamil.pokornicki@3mdeb.com](mailto:kamil.pokornicki@3mdeb.com)

### CODE REVIEW

File: docs/variants/novacustom_ns5x/hardware-matrix.md [hunk 1]
```diff
@@ -55,5 +56,6 @@ coreboot port on the Clevo NS51/NS70 laptops.
 |                                | 5. SanDisk Ultra microSD HC I 16 GB + SD Adapter |
 |                                | 6. Logitech, Inc. Keyboard K120                  |
 |                                | 7. 4-port USB Type-C Hub Pro                     |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/novacustom_ns5x/hardware-matrix.md:58]: In my opinion, there's no sense in putting two different USB-C hubs in one HW matrix. Please remove from the table `4-port USB Type-C Hub Pro`.

File: docs/variants/clevo_nv41/hardware-matrix.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ coreboot port on the Clevo NV41 laptop.
 |                                | 5. Logitech, Inc. Keyboard K120                  |
 |                                | 6. 4-port USB Type-C Hub Pro                     |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/clevo_nv41/hardware-matrix.md:29]: In my opinion, there's no sense in putting two different USB-C hubs in one HW matrix. Please remove from the table `4-port USB Type-C Hub Pro`.

File: docs/variants/clevo_nv41/hardware-matrix.md [hunk 3]
```diff
@@ -56,6 +57,7 @@ coreboot port on the Clevo NV41 laptop.
...
```
Comment (sulewskiprzemyslaw) [inline docs/variants/clevo_nv41/hardware-matrix.md:58]: In my opinion, there's no sense in putting two different USB-C hubs in one HW matrix. Please remove from the table `4-port USB Type-C Hub Pro`.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/novacustom_ns5x/hardware-matrix.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/variants/novacustom_ns5x/hardware-matrix.md]: removed

File: docs/variants/clevo_nv41/hardware-matrix.md (hunk 3 above)
Comment (Pokisiekk) [reply docs/variants/clevo_nv41/hardware-matrix.md]: removed

**Pull request merged**